.github/workflows/bot_fix_review.yaml

```diff
@@ -0,0 +1,102 @@
+name: bot fix style (review)
+
+on:
+  pull_request_review:
+    # triggers on a review, whether or not it is accompanied by a comment
+    types: [submitted]
+
+jobs:
+  fix_style:
+    name: Fix style issues from lint
+    if: (startsWith(github.event.review.body, 'bot fix style') || contains(toJSON(github.event.review.body), '\nbot fix style'))
+    runs-on: ubuntu-latest
+    continue-on-error: true  # do not annoy the user when linting fails, as expected
+    steps:
+      - id: user_permission
+        uses: actions-cool/check-user-permission@v2
+        with:
+          require: 'write'
+
+      - run: echo ${{ steps.user_permission.outputs.require-result }}
+
+      # Maybe no API exists for this yet?
```

Comment:
Is there a way to add a reaction to a PR review via the API? I tried the following and it gave me a 404 (I'm pretty sure the token I used has the permissions necessary): https://github.com/bryangingechen/mathlib4/actions/runs/10534530644/job/29192315609#step:4:14